labs-ios-starter/Helpers/ParseCities.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ParseCities.swift
+//  labs-ios-starter
+//
+//  Created by Conner on 1/24/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+// Parse cities from CSV file
```

Comment:
This is really neat, to include a helper file to only get cities.

Reply:
Very nifty, looks like it'll work great to filter just cities